README.md

```diff
@@ -109,12 +109,6 @@ _Default value_: `process.cwd()`
 
 Path to the website's directory.
 
-#### ignoredWatchCommand
-
-_Type_: `string`
-
-When detecting the watch command, ignore `package.json` `scripts` whose value includes this string.
```

Comment:
Should we remove this in a separate PR?

Reply:
Sure I'll create a separate PR to remove it